pkg/controller/provider/container/ovirt/resource.go

```diff
@@ -87,13 +94,32 @@ func (r *Cluster) ApplyTo(m *model.Cluster) {
 	m.HaReservation = r.bool(r.HaReservation)
 	m.KsmEnabled = r.bool(r.KSM.Enabled)
 	m.BiosType = r.BiosType
+	m.CPU.Type = r.CPU.Type
+	m.Version.Minor = r.Version.Minor
+	m.Version.Major = r.Version.Major
 }
 
 // Cluster (list).
 type ClusterList struct {
 	Items []Cluster `json:"cluster"`
 }
 
+// ServerCpu.
+type ServerCpu struct {
+	Base
+	Values struct {
+		SystemOptionValues []struct {
```

Comment:
isn't it possible to do something like:
```suggestion
		SystemOptionValues []SystemOptionValue `json:"system_option_value"`
```